ml_metadata/tools/mlmd_bench/stats.h

```diff
@@ -20,13 +20,55 @@ limitations under the License.
 
 namespace ml_metadata {
 
-// OpStats records the statics(elapsed microsecond, transferred bytes) of each
+// OpStats records the statics(elapsed time, transferred bytes) of each
 // operation. It will be used to update the thread stats.
 struct OpStats {
   absl::Duration elapsed_time;
   int64 transferred_bytes;
 };
 
+class Stats {
```

Comment:
nit, consider to call it ThreadStats, probably is more clear. Please add some class docstring to describe its usage.

Reply:
Sounds good! Fixed!